src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -30,7 +30,18 @@
                 </form>
                 <filters></filters>
             </div>
-            <div class="col-md-9 sap-filters">
+            <div class="col-md-9">
+                <ul class="nav nav-tabs">
+                    <li class="nav-item">
+                        <a class="nav-link" :class="survey.tabClass" v-on:click="selectTab(2)">Survey</a>
```

Comment:
Note the calls to e.g. `selectTab(2)` - these have to be js rather than ts, hence not being able to use the enum. 

Alternatives: 
- create a js enum instead of a ts one (i.e. an object rather than a type)
- define 3 separate component methods "selectSurveyTab" etc. which can use the enum, and call them

I'm inclined to leave as is given it's the simplest solution and the tab changing functionality is under test